capstone/src/main/java/com/google/sps/data/ReadData.java

```diff
@@ -5,51 +5,101 @@
 import com.google.appengine.api.datastore.Entity;
 import com.google.appengine.api.datastore.EntityNotFoundException;
 import java.io.BufferedReader;
+import java.io.FileNotFoundException;
 import java.io.FileReader;
 import java.io.IOException;
 import java.text.ParseException;
 import java.text.SimpleDateFormat;
+import java.util.ArrayList;
 import java.util.Date;
+import java.util.Set;
 
 public class ReadData {
-  final String DELIMITER = ",";
+  private DatastoreService datastoreService;
+  private SentimentData sentimentService;
+  private ArrayList<Entity> newEntities;
+  private final String DELIMITER = ",";
+
+  public ReadData() throws IOException {
+    datastoreService = DatastoreServiceFactory.getDatastoreService();
+    sentimentService = new SentimentData();
+    newEntities = new ArrayList<Entity>();
+  }
+
+  /**
+   * Calls read file while iterating through all the zip codes
+   *
+   * @return {Void}
+   */
+  public void readAll() {
+    Set<String> zipCodes = MapData.zipPrecinctMap.keySet();
+
+    for (String zip : zipCodes) {
+      String filePath = "assets/" + zip + ".csv";
+
+      try {
+        readFile(new FileReader(filePath), zip);
+      } catch (FileNotFoundException e) {
+        System.out.println(filePath + " is not in the assets folder");
+      }
+    }
+
+    sentimentService.close();
+  }
 
   /**
-   * Reads the survey csv and adds each element to datastore
+   * Reads the survey csv, splits the lines by a comma delimiter, and adds each element to data
+   * store as an entity property
    *
+   * @param file This is the file that is being parsed
+   * @param zip This is the zip code the file is associated with
    * @return {Void}
    */
-  public void readCSV() {
+  public void readFile(FileReader file, String zip) {
+    final int EXPECTED_LENGTH = 15;
     BufferedReader reader = null;
-    FileReader file = null;
-    DatastoreService datastoreService = DatastoreServiceFactory.getDatastoreService();
 
     try {
-      file = new FileReader("assets/surveyresponse.csv");
       reader = new BufferedReader(file);
       String header = reader.readLine();
       String line = "";
 
       while ((line = reader.readLine()) != null) {
         String[] values = line.split(DELIMITER);
-        String id = values[0] + "-" + values[1];
+        String id = values[0];
         SimpleDateFormat formatter = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss");
         Date date;
 
         try {
-          date = formatter.parse(values[2]);
+          date = formatter.parse(values[1]);
         } catch (ParseException e) {
           date = null;
         }
-        String completion = values[3];
-        String gender = values[5];
-        String ageRange = values[6];
+
+        String completion = values[2];
+        String gender = values[4];
+        String ageRange = values[5];
         String answerOne = values[8];
```

Comment:
Same comment as before with the naming here.

Reply:
+1

Reply:
I will make these changes in my next PR